src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.WaitThread.cs

```diff
@@ -20,23 +21,26 @@ internal partial class PortableThreadPool
         /// <param name="handle">A description of the requested registration.</param>
         internal void RegisterWaitHandle(RegisteredWaitHandle handle)
         {
+            PortableThreadPoolEventSource log = PortableThreadPoolEventSource.Log;
+            if (log.IsEnabled())
```

Comment:
Changed to use `PortableThreadPoolEventSource.Log` instead of loading the log into a variable. I figured the load of the log would be unused and removed when `IsEnabled()` is statically false, not sure.